src/SfxWeb/src/app/modules/repair-jobs/repair-tasks/repair-tasks.component.html

```diff
@@ -108,7 +108,7 @@ <h4 collapse-header>
     </div>
 
 
-    <div class="detail-pane essen-pane" data-cy="pendingjobs">
+    <div class="detail-pane essen-pane" data-cy="pendingjobs" *ngIf="isShown">
```

Comment:
didnt we want to still show the list of repair jobs?

Reply:
no, the idea was to just show the chart and not the list